### ISSUE TITLE
Pass context to ready and after

### ISSUE THREAD

**Issue by mcdonnelldean** [mcollina/avvio#6]
This allows these functions to be written in other modules. consider the following.

``` js
// some setup function that handles post plugin
// loading for some icky plugin.
module.exports = function (context, done) {
  // I can use the context without issue now
  context.dispatch('comms:open')
  done()
}

```

**Comment by mcollina** [comment]
Yeah! PR?

**Comment by mcdonnelldean** [comment]
It's on the list :D

**Comment by mcollina** [comment]
Fixed in v0.5.0.
